bindings/matrix-sdk-ffi/src/timeline.rs

```diff
@@ -1252,3 +1286,15 @@ impl From<&matrix_sdk_ui::timeline::AnyOtherFullStateEventContent> for OtherStat
         }
     }
 }
+
+#[derive(Clone, uniffi::Enum)]
+pub enum PollKind {
+    Disclosed,
+    Undisclosed,
+}
+
+#[derive(Clone, uniffi::Record)]
+pub struct PollAnswer {
+    pub id: String,
+    pub text: String,
+}
```

Comment:
Is it the code from https://github.com/matrix-org/matrix-rust-sdk/pull/2326? Can we close #2326 in favor of this PR?

Reply:
See PR description (in particular todo list at the end) 😉

Reply:
/me whistles